src/verify_expr.ml

```diff
@@ -503,30 +504,35 @@ module VerifyExpr(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
   
   let interf_specs_for_sign map1 sign (itf, passedTypes) =
     let InterfaceInfo (_, fields, meths, _, _, tparams) = List.assoc itf map1 in
-    let interTparamEnv = List.map2 (fun a b -> (a, b)) tparams passedTypes in
+    (* If we have tparams, but no passed types: raw type *)
+    let interTparamEnv = match tparams with 
+      [] -> [] 
+    | tparams when passedTypes = [] -> List.map (fun tparam -> (tparam, javaLangObject)) tparams
+    | tparams -> List.map2 (fun a b -> (a, b)) tparams passedTypes
+    in

```

Comment:
Kan dit voorkomen? Ik dacht dat raw types in `check_pure_type` al omgezet werden naar parameterized types?

Reply:
Dit zit nog niet in de laatste pull, ik bekijk het nu